includes/abstracts/abstract-wp-job-manager-form.php

```diff
@@ -603,7 +605,7 @@ protected function upload_file( $field_key, $field ) {
 			}
 
 			$file_urls       = array();
-			$files_to_upload = job_manager_prepare_uploaded_files( $_FILES[ $field_key ] );
+			$files_to_upload = job_manager_prepare_uploaded_files( $_FILES[ $field_key ] ); //phpcs:ignore WordPress.Security.ValidatedSanitizedInput
```

Comment:
See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1720 for context

Reply:
When ignoring, use the full `Standard.Category.SniffName.ViolationCode`, not just `Standard.Category.SniffName`. Otherwise you might be ignoring other checks that are in / added to the same sniff.

Reply:
Add this PR comment as a reason on the ignore:

```
//phpcs:ignore WordPress.Security.ValidatedSanitizedInput - See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1720.
```

Reply:
We need to ignore both the Unslash and Sanitization here. I couldn't find where to ignore multiple rules on the same line. Do you know how?

Reply:
Just use comma separation:

```
// phpcs:ignore A.B.C.D, A.B.C.E
```

(or maybe remove the space after the comma if that doesn't work)

Reply:
That works! Very long line.